### ISSUE TITLE
Introduce a verbose/list option to the build command

### ISSUE THREAD

**Issue by sagikazarmark** [markbates/pkger#104]
In the output of a docker build command it would be nice to enable a verbose mode to list the files added to the bundle. It would help with debugging.

**Comment by phanirithvij** [comment]
Yes, running `pkger list` before `pkger` doubles the build time. Please add `pkger -v`
